internal/app/api/api.go

```diff
@@ -291,7 +291,8 @@ func (a *Client) do(doer HTTPDoer) (*http.Response, error) {
 // GetMedia fetches the media object from iAuditor.
 func (a *Client) GetMedia(ctx context.Context, request *GetMediaRequest) (*GetMediaResponse, error) {
 	baseURL := strings.TrimPrefix(request.URL, a.baseURL)
-	mediaID := strings.TrimPrefix(baseURL, fmt.Sprintf("/audits/%s/media/", request.AuditID))
+	mediaIDURL := strings.Split(request.URL, "/")
+	mediaID := mediaIDURL[len(mediaIDURL)-1]
```

Comment:
Can we add a comment please, so future folks don't have to guess at what the expected format of the media url looks like?